pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -281,7 +283,11 @@ func (ctx *context) handleInboundInvitation(invitation *Invitation, thid string)
 		return nil, err
 	}
 	// choose the first public key
-	sendVerKey := string(newDidDoc.PublicKey[0].Value)
+	pubKey, err := getPublicKey(newDidDoc, supportedPublicKeyType)
+	if err != nil {
+		return nil, err
+	}
+	sendVerKey := string(pubKey[0].Value)
```

Comment:
why pubKey[0]?

Reply:
so did document can have a multiple keys for same type. Therefore we will pick the first one.

Reply:
Is this the documented behaviour in the protocol spec?